cvat/apps/engine/serializers.py

```diff
@@ -1030,6 +1030,8 @@ def _create_files(self, instance, files):
                 )
 
 class StorageSerializer(serializers.ModelSerializer):
+    cloud_storage_id = serializers.IntegerField(required=False, allow_null=True)
+
     class Meta:
         model = models.Storage
         fields = ('id', 'location', 'cloud_storage_id')
```

Comment:
`cloud_storage_id` should probably be removed from here, since it's not a field anymore.

Reply:
I don't think so.
https://www.django-rest-framework.org/api-guide/serializers/#specifying-fields-explicitly
```
AssertionError: The field 'cloud_storage_id' was declared on serializer StorageSerializer, but has not been included in the 'fields' option.
```